client/src/pages/Landing/index.js

```diff
@@ -15,6 +16,13 @@ const useStyles = makeStyles((theme) => ({
 
 const Landing = () => {
 	const classes = useStyles();
+	const { user, isAuthenticated } = useAuth0();
+
+	useEffect(() => {
```

Comment:
This code in here will need to be in whatever landing page exists in order for user authentication to work.